mirdata/beatles.py

```diff
@@ -123,6 +127,20 @@ def to_jams(self):
         )
 
 
+def load_audio(audio_path):
```

Comment:
couldn't this function be shared across datasets, in most cases? like, `from utils import load_audio`; and if not, implement it inside the dataset-specific module

Reply:
Yeah I was thinking the same. I realize it's redundant, but it also feels sort of silly to have a common function that just wraps librosa, which would need to have the same parameters as the librosa function to be generically useful. Right now I'm writing very specific wrappers per dataset on the librosa function that hard codes certain parameters. What do you think?

Reply:
i'm not sure. i would say: feel free to carry on with your WIP PR and we'll advise later. i can live with it.